.github/workflows/automatic-doc-checks.yml

```diff
@@ -0,0 +1,49 @@
+name: Main Documentation Checks
```

Comment:
why do we have this if we have similar tasks in our tox driven workflow?

@lengau?

Reply:
Per separate discussion: https://github.com/canonical/craft-providers/issues/420